poc-step-function/lambda-functions/FetchAssetMetadata/index.js

```diff
@@ -0,0 +1,45 @@
+// This fakes reading data from S3 and just returns the expected metadata.
```

Comment:
Instead of actually retrieving this from S3, this example just spits out an example structure of the metadata.